app/frontend/src/features/connections/ConnectionsPage.tsx

```diff
@@ -3,24 +3,27 @@ import React from "react";
 import { useHistory, useParams } from "react-router";
 import { connectionsRoute } from "../../AppRoutes";
 import PageTitle from "../../components/PageTitle";
-import FriendsTab from "./friends/FriendsTab";
+import { FriendsTab } from "./friends";
 import TabBar from "../../components/TabBar";
 
 const labels = {
   all: "All",
   friends: "Friends",
 };
 
+type ConnectionType = keyof typeof labels;
+
 function ConnectionsPage() {
   const history = useHistory();
-  const { type = "all" } = useParams<{ type: keyof typeof labels }>();
+  const { type = "all" } = useParams<{ type: string }>();
+  const connectionType = type in labels ? (type as ConnectionType) : "all";
```

Comment:
This puts the user to the "all" tab by default if they type in some gibberish in the URL (e.g. `/connections/blah`), which I thought it's a more sensible default and less odd than not having a tab selected at all when the user is on the connections page.